org/pr/installable-build.ts

```diff
@@ -136,7 +138,7 @@ async function getDownloadCommentText(status) {
 }
 
 export default async (status: Status) => {
-    if (status.state == "pending" && HOLD_CONTEXTS.includes(status.context)) {
+    if (status.state == "pending" && HOLD_CONTEXTS.concat(HOLD_CONTEXTS_NO_COMMENT).includes(status.context)) {
       await markStatusAsSuccess(status)
       await createOrUpdateComment(status, `You can trigger an installable build for these changes by visiting CircleCI [here](${status.target_url}).`)
     } else if (status.state ==  "success" && INSTALLABLE_BUILD_CONTEXTS.filter(s => status.context.match(s)).length > 0) {
```

Comment:
If I'm not mistaken, this is the only place where both `HOLD_CONTEXTS` and `HOLD_CONTEXTS_NO_COMMENTS` constants are used.

 - If that's intended, and the only place where they are used here they are concatenated, not sure what's the point of having them split in the first place?
 - Instead I think what you intended to do is to NOT call `createOrUpdateComment` for the case of `HOLD_CONTEXTS_NO_COMMENTS`?

Reply:
Oh, I didn't notice my last push failed 🤦! I guess this is what you get when you try to write some code while you have to respond to too many pings 😂

Reply:
Yeah! That version was clearly doing nothing :-)